src/main/java/com/example/demo/PatientController.java

```diff
@@ -64,6 +64,23 @@ public Patient editPatientByEmail(@PathVariable String email, @RequestBody Patie
         existingPatient.get().setPhoneNumber(editedPatient.getPhoneNumber());
         return editedPatient;
     }
+
+    @PatchMapping("/{email}/password")
+    public String updatePassword(@PathVariable String email, @RequestBody String newPassword) {
+        Optional<Patient> existingPatient = findPatientByEmail(email);
```

Comment:
.orElseThrow(() -> throw new IllegalArgumentException("Patient with given email does not exist"));

Reply:
Patient existingPatient = findPatientByEmail(email)
                                       .orElseThrow(() -> throw new IllegalArgumentException("Patient with given email does not exist"));
                                       
                                       WTEDY IF DO USUNIECIA